build.sh

```diff
@@ -312,6 +356,10 @@ function main {
         build_tssim
     elif [ "$1" == "sysrel" ]; then
         build_sysrel
+    elif [ "$1" == "frontend" ]; then
+        build_frontend
+    elif [ "$1" == "prepare-riesgos-wps" ]; then
+        prepare_riesgos_wps
```

Comment:
```suggestion
```

There is no prepare_riesgos_wps anymore, as this is part of the init wps container.

Reply:
Right! Being removed.